### DISCUSSION TITLE
Is it possible to navigate to pages in a website automatically ?

### DISCUSSION THREAD

**Discussion by rthangam** [GoogleChrome/lighthouse-ci#558] [General]
I am able to use puppeteer to login and test my urls, but is it possible to navigate to all possible pages of an application  ?

**Comment by patrickhulce** [comment]
Could you clarify your use case a bit? Puppeteer can of course navigate around your site, but I'm not sure how that ties in with LHCI and what you're hoping to accomplish.

**Reply by rthangam** [reply]
Is there a way Puppeteer can find all possible links or pages in a site it can navigate to or we need to specify the urls in configuration only ?

**Reply by patrickhulce** [reply]
You will need to specify the URLs in the configuration.

**Reply by patrickhulce** [reply]
If your site is static, you can use the autodiscover features of LHCI which will automatically run on every `.html` file in your static output folder.

**Reply by rthangam** [reply]
We have a website to run on, can it still autodiscover ?

**Reply by patrickhulce** [reply]
No.
